src/main/scala/nl.knaw.dans.easy.deposit/authentication/TokenSupport.scala

```diff
@@ -56,7 +59,7 @@ trait TokenSupport extends DebugEnhancedLogging {
   def decodeJWT(token: String): Try[AuthUser] = {
     for {
       decoded <- Jwt.decode(token, tokenConfig.secretKey, Seq(tokenConfig.algorithm))
-      parsed <- fromJson(decoded)
+      parsed <- fromJson(decoded.toJson)
```

Comment:
in the toJson I see

    "iss" -> issuer,
    "sub" -> subject,
    "aud" -> audience.map(set => if (set.size == 1) set.head else set),
    "exp" -> expiration,
    "nbf" -> notBefore,
    "iat" -> issuedAt,
    "jti" -> jwtId

Which should parse as `case class Token(exp: Long, iat: Long, uid: String)` 
I can match exp/iat but not uid, otherwise the the detour could be rewritten as
`Jwt.decode(...).map(claim => Token(claim.exp, claim.iat, claim.???))`